crowdin.yml

```diff
@@ -0,0 +1,41 @@
+preserve_hierarchy: true
+
+files:
+  - source: /episodes/*.Rmd
+    dest: /episodes/%file_name%.md
```

Comment:
@joelnitta this lesson uses R Markdown source files for its episodes. Should the extension here on line 5 be updated to `.Rmd`?

Reply:
Yes, please change that.